man/sd_notify.xml

```diff
@@ -493,6 +493,11 @@
     privileged port (i.e.: lower than 1024), as an attempt to address concerns that unprivileged processes in
     the guest might try to send malicious notifications to the host, driving it to make destructive decisions
     based on them.</para>
+
+    <para>Note that, while using this library should be preferred in order to avoid code duplication, it is
```

Comment:
I thought, based on https://mastodon.social/@pid_eins/112202687764571433, using this library just for notify is not recommended?